composer.json

```diff
@@ -6,6 +6,7 @@
 	],
 	"require": {
 		"php": "^7.4 || ^8.0",
+		"bartlett/sarif-php-sdk": "^1.0",
```

Comment:
Please try to do this without a dependency. It's just encoding a simple JSON after all.